src/js/components/ChatTop/ChatTop.js

```diff
@@ -2,28 +2,87 @@ import PropTypes from 'prop-types';
 import React from 'react';
 import '../../../scss/components/ChatTop/ChatTop.scss';
 
+const topButtons = [
+  {
+    iconClass: 'svg svg-voice-call',
+    tip: 'Start Voice Call',
```

Comment:
I recognise that we display this in a tooltip, but is there a better property for it? Maybe `title`?